src/utils/mapLinks.js

```diff
@@ -103,74 +89,60 @@ export const getMapUrl = (buildingRoom, instructor) => {
       break
     case 'GHC':
     case 'GRAHAM HEALTH CENTER':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71370'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71370'
       break
     case 'OVH':
     case 'OAKVIEW HALL':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71390'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71390'
       break
     case 'VBH':
     case 'WVH':
     case 'VANDENBERG HALL':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/65256'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/65256'
       break
     case 'HAM':
     case 'HAMLIN HALL':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/65257'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/65257'
       break
     case 'VWH':
     case 'VAN WAGONER HOUSE':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/73413'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/73413'
       break
     case 'HIL':
     case 'HILL HOUSE':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/65258'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/65258'
       break
     case 'FTZ':
     case 'FITZGERALD HOUSE':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71387'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71387'
       break
     case 'ANI':
     case 'ANIBAL HOUSE':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71388'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71388'
       break
     case 'PRY':
     case 'PRYALE HOUSE':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71389'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71389'
       break
     case 'JDH':
     case 'JOHN DODGE HOUSE':
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71402'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71402'
       break
     default:
-      url =
-        'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600'
+      url = 'https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600'
```

Comment:
Do you think there could be a better way to store all these map links? Maybe in a JSON object where the key is the name of the building and the value is the link? That way you could just do something like `url = locations[location]`.

I see that there are duplicate values that correspond to each map link. You could add those into the JSON twice or have some other object that maps the abbreviated building strings to the full name or something. It could look something like this:
```js
let abbreviations = {
"JDH": "JOHN DODGE HOUSE"
}

let mapLinks = {
"JOHN DOGE HOUSE": "https://www.myatlascms.com/map/index.php?id=566#!ct/5468,6596,6597,6598,6600?m/71402"
}

// ...

location = abbreviations.hasOwnProperty(location) ? abbreviations[location] : location
let url = mapLinks[location]
```
This isn't a show stopper for me but I don't think it would hurt if this section were simplified a bit without having to write out all the logic. I'm interested in how others would feel about this change.

Reply:
Yeah, I know that @amgoodfellow mentioned something about updating broken/new links, maybe whoever does it can also think of a better way to go about it. I don't know if people want that to be done in this PR or a follow up issue tho

Reply:
Yeah, that's a lot of URLs. I don't think you need to resolve it in this PR, but we might even want to explore an external source for these, something that made goes out and gets this JSON and can be updated without having to cut a new release. Maybe storying all of this in Postgres and having a single call to get the JSON of the key/values. :man_shrugging:

Reply:
I think we could all have a discussion about this to see what ideas others may have